src/app/shared/services/user-session.service.ts

```diff
@@ -40,6 +40,10 @@ export class UserSessionService implements OnDestroy {
     return !!this.session$.value?.watchedGroup;
   }
 
+  get userGroupId(): string | undefined {
```

Comment:
probably not useful anymore with the previous remark.

Reply:
Fixed